docs/csharp/getting-started/library-with-visual-studio.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Building a class library with C# and .NET Core in Visual Studio 2017
+description: Learn how to build a class library written in C# using Visual Studio 2017
+keywords: .NET Core, .NET Standard class library, Visual Studio 2017
+author: stevehoag
+ms.author: shoag
+ms.date: 03/07/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: c849ca26-6a25-4d35-9544-f343af88e0e7
+---
+
+# Building a class library with C# and .NET Core in Visual Studio 2017
+
+A *class library* defines types and methods that are called by an application. A class library developed using .NET Core supports the .NET Standard Library, which allows your library to be called by any .NET platform that supports that version of the .NET Standard Library. When you finish your class library, you can decide whether you want to distribute it as a third-party component or whether you want to include it as a bundled component with one or more applications.
+
+> [!NOTE]
+> For a list of the .NET Standard versions and the platforms they support, see [.NET Standard Library](../../standard/library.md).
+
+In this topic, you'll create a simple utility library that contains a single string-handling method. You'll implement it as an [extension method](../../csharp/programming-guide/classes-and-structs/extension-methods.md) so that you can call it as if it were a member of the @System.String class.
+
+## Creating a class library solution
+
+Start by creating a solution for your class library project and its related projects. A Visual Studio Solution just serves as a container for one or more projects. To create the solution:
+
+1. On the Visual Studio menu bar, choose **File > New > Project**.
+
+1. In the **New Project** dialog, expand the **Other Project Types** node, and select **Visual Studio Solutions**. Name the solution "ClassLibraryProjects" and select the **OK** button.
```

Comment:
I was trying to follow the existing pattern where if the user is directed to type something the text is in quotes. In other places, the project name is in-line code.